lib/aida/db/db.ex

```diff
@@ -375,7 +375,9 @@ defmodule Aida.DB do
 
   Returns `nil` if the Image does not exist.
   """
-  def get_image(id), do: Repo.get_by(Image, uuid: id)
+  def get_image(id) do
+    Repo.get_by(Image, uuid: id) || {:error, :not_found}
```

Comment:
If this method returns a tuple when it doesn't find the image, it should return a tuple when it finds it, like {:ok, image}